src/Layouts/LayersList/LayerItemModel.vala

```diff
@@ -39,7 +39,13 @@ public class Akira.Layouts.LayersList.LayerItemModel : GLib.Object {
      */
     public string name {
         owned get {
-            return node.id.to_string (); // Temporarily use the ID.
+            return node.components.name.name;
+        }
+        set {
+            if (value == name) {
+                return;
+            }
+            node.components.name = new Lib.Components.Name (value, id.to_string ());
```

Comment:
@mbfraga is this correct?
I feel like it doesn't follow the approach other components are replaced as I'm changing it directly inside the Layer Model, which is its own thing separated from the Item Model, but I thought it was fine since the name can't be really edited from within the canvas without a layer UI.

Reply:
Can you point me to where in the current code we do something like that?

Reply:
This? https://github.com/akiraux/Akira/blob/main/src/Utils/AffineTransform.vala#L80-L82